shuup/admin/modules/media/views.py

```diff
@@ -257,7 +256,11 @@ def handle_get_folder(self, data):
             else:
                 folder = None
                 subfolders = _get_folder_query(shop, self.user).filter(parent=None)
```

Comment:
This should most likely go inside the if-block as well to get the same treatmeant as files.